segmentation/mesmer-inference-plugin/src/polus/plugins/segmentation/mesmer_inference/__main__.py

```diff
@@ -0,0 +1,100 @@
+"""Mesmer Inference."""
+import json
+import logging
+import pathlib
+from typing import Any, Optional
+
+import filepattern as fp
+import typer
+
+from polus.plugins.segmentation.mesmer_inference.padded import Extension, Model, run
+
+# Initialize the logger
+logging.basicConfig(
+    format="%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s",
+    datefmt="%d-%b-%y %H:%M:%S",
+)
+logger = logging.getLogger("polus.plugins.segmentation.mesmer_inference")
+logger.setLevel(logging.INFO)
```

Comment:
This should grab the log level from the environment variable, and default to `logging.INFO` if it's not present.

Reply:
Done